src/Test/UnitTests.cs

```diff
@@ -55,6 +56,21 @@ public async Task ProxyAttributePostRequest()
             Assert.Contains("101", JObject.Parse(responseString).Value<string>("id"));
         }
 
+        [Fact]
+        public async Task ProxyAttributePostWithFormRequest()

```

Comment:
Thanks for adding a test! :)

Reply:
Tests are included in project so they should be updated also ;)